output/transform/collector_platform.go

```diff
@@ -0,0 +1,22 @@
+package transform
+
+import (
+	"github.com/golang/protobuf/ptypes"
+	snapshot "github.com/pganalyze/collector/output/pganalyze_collector"
+	"github.com/pganalyze/collector/state"
+)
+
+func transformCollectorPlatform(s snapshot.FullSnapshot, state state.TransientState) snapshot.FullSnapshot {
+	p := state.CollectorPlatform
+	startTs, _ := ptypes.TimestampProto(p.StartedAt)
```

Comment:
I noticed we're ignoring the `ptypes.TimestampProto` error elsewhere, so doing that here, too--lmk if we should be doing something with this.

Reply:
I *think* we're okay unless something is totally off the rails: https://github.com/golang/protobuf/blob/master/ptypes/timestamp.go#L80-L88

Reply:
Yeah, its safe to ignore these in my opinion